stride/assetlist.json

```diff
@@ -0,0 +1,114 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "stride",
+    "assets": [
+      {
+        "description": "The native token of Stride",
+        "denom_units": [
+          {
+            "denom": "ust",
+            "exponent": 0,
+            "aliases": []
+          },
+          {
+            "denom": "st",
+            "exponent": 6,
+            "aliases": []
+          }
+        ],
+        "base": "ust",
+        "name": "Stride",
+        "display": "st",
+        "symbol": "ST",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/st.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/st.svg"
+        },
+        "coingecko_id": "stride"
+      },
+      {
+        "denom_units": [
+          {
+            "denom": "ustatom",
+            "exponent": 0
+          },
+          {
+            "denom": "statom",
+            "exponent": 6
+          }
+        ],
+        "base": "ustatom",
+        "name": "stATOM",
+        "display": "statom",
+        "symbol": "stATOM",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/statom.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/statom.svg"
+        },
+        "coingecko_id": "statom"
+      },
+      {
+        "denom_units": [
+          {
+            "denom": "ustatom",
+            "exponent": 0
+          },
+          {
+            "denom": "statom",
+            "exponent": 6
+          }
+        ],
+        "base": "ustatom",
+        "name": "stATOM",
+        "display": "statom",
+        "symbol": "stATOM",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/statom.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/statom.svg"
+        },
+        "coingecko_id": "statom"
```

Comment:
not a CGID in https://api.coingecko.com/api/v3/coins/list
is it supposed to be "stride-staked-atom"?